front/pages/api/w/[wId]/assistant/agent_configurations/[aId]/index.ts

```diff
@@ -75,7 +76,13 @@ async function handler(
   switch (req.method) {
     case "GET":
       return res.status(200).json({
-        agentConfiguration: assistant,
+        agentConfiguration: {
+          ...assistant,
+          lastAuthors: await getAgentRecentAuthors({
```

Comment:
Don't we want to gate this behind a query parameter?

Reply:
don't think so, since we already retrieve the "full" variant here ( + the call to recent authors is lighter now, and cached) => there is no real benefit to add code for this IMHO

Reply:
Yes please this route is crirical for app speed

Reply:
yes, 1 agent full route